prow/plank/report.go

```diff
@@ -17,18 +17,19 @@ limitations under the License.
 package plank
 
 import (
+	"bytes"
 	"fmt"
 	"strings"
+	"text/template"
 
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plugins"
 )
 
-const (
-	commentTag  = "<!-- test report -->"
-	guberPrefix = "https://k8s-gubernator.appspot.com/pr/"
-)
+const commentTag = "<!-- test report -->"
+
+var tmpl = template.Must(template.New("report").Parse(`[Full PR test history](https://k8s-gubernator.appspot.com/pr/{{if eq .Spec.Refs.Org "kubernetes"}}{{if eq .Spec.Refs.Repo "kubernetes"}}{{else}}{{.Spec.Refs.Repo}}/{{end}}{{else}}{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{end}}{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}). [Your PR dashboard](https://k8s-gubernator.appspot.com/pr/{{with index .Spec.Refs.Pulls 0}}{{.Author}}{{end}}). Please help us cut down on flakes by [linking to](https://github.com/kubernetes/community/blob/master/contributors/devel/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/{{.Spec.Refs.Org}}/{{.Spec.Refs.Repo}}/issues?q=is:issue+is:open) when you hit one in your PR.`))
```

Comment:
One way to work around this annoyance is to just get rid of this logic from the template, always do `org_repo`, and have gubernator turn `'k_k/'` into `''` instead.

I actually strongly dislike this whole scheme with shortening kubernetes URLs. It's pretty confusing, and we have the same code duplicated in all of our tools to deal with it. I'm pretty sure we only did it initially because that's where we were already uploading logs. It actually probably won't be too tricky to fix, actually.